client/src/elements/item-preview/item-preview.js

```diff
@@ -109,10 +109,10 @@ export class ItemPreview {
       // set the default preview width to the most narrow variant
       this.previewWidthProxy.width = this.sizeOptions[0].value;
 
-      this.availableTargets = await this.qTargets.get("availableTargets");
+      const availableTargets = await this.qTargets.get("availableTargets");
```

Comment:
I didn't see any reason why those values should be exposed on the component itself.

Reply:
Yeah, that is most probably from back in the days where we had a switcher in the preview UI where users could switch between targets.
Since that switcher is gone, you change is only consequent.